public/javascripts/modals.js

```diff
@@ -37,6 +37,10 @@ var modalsNS = odkmaker.namespace.load('odkmaker.modals');
             {
                 $dialog.find('.modalLoadingOverlay').stop().fadeOut();
 
+                odkmaker.auth.currentUser.forms.sort(function(a, b) {
```

Comment:
Would it make sense to put the length check before doing anything with the list?

Reply:
I think this is fine as-is; an array is guaranteed and sorting and empty array is a free no-op.